### DISCUSSION TITLE
Benefits not available

### DISCUSSION THREAD

**Discussion by Ompro24** [community/community#137173] [GitHub Education] [answered]
### Hi everyone,

Benefits are unavailable for me could you please help me to resolve the problem I have been approved on 28th Aug

**Comment by davevad93** [comment, marked as answer]
Hi @Ompro24 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ompro24 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by doyalsnehil** [comment]
same here , request was approved on 28th

**Comment by webdebug** [comment]
The application was also approved for me on 28.08.2024 and the benefits are still not available.
